resources/js/components/views/members/form.tsx

```
@@ -0,0 +1,165 @@
+} from '@/components/ui/field';
+import { Input } from '@/components/ui/input';
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from '@/components/ui/select';
+import { Spinner } from '@/components/ui/spinner';
+import MembersController from '@/wayfinder/App/Http/Controllers/Units/MembersController';
+import { App } from '@/wayfinder/types';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(`@typescript-eslint/consistent-type-imports`)

---

[error] `@inertiajs/react` import should occur before import of `@/components/ui/button`

(import/order)